Matlab/Matlab.sublime-syntax

```diff
@@ -48,6 +48,10 @@ contexts:
       scope: comment.line.percentage.matlab
       captures:
         1: punctuation.definition.comment.matlab
+    - match: (\.{3}).*$\n?
+      scope: comment.line.ellipsis.matlab
+      captures:
+        1: punctuation.definition.comment.matlab
```

Comment:
I would suggest an implementation with something like the following:
```yaml
- match: (\.{3})(.*$\n?)
  captures:
    1: punctuation.separator.continuation.line.matlab
    2: comment.line.ellipsis.matlab
```

This would allow the `...` to not be rendered as comment, because its main purpose is line continuation. The [official .tmLanguage grammar](https://github.com/mathworks/MATLAB-Language-grammar/blob/40d9a0cd3b628f80cdcf948bbe1747a527ed5dd5/Matlab.tmbundle/Syntaxes/MATLAB.tmLanguage#L1375-L1396) maintained by Mathworks uses `keyword.operator.symbols` for it, but I think `punctuation.separator.continuation.line` is more consistent with the usage in other default syntaxes here.

Reply:
Reusing the existing continuation pattern.